shipyard_blueprints/snowflake/tests/test_auth.py

```diff
@@ -1,11 +1,13 @@
 import os
 from shipyard_snowflake import SnowflakeClient
 from dotenv import load_dotenv, find_dotenv
+from shipyard_snowflake import SnowparkClient
+from typing import Union
 
 load_dotenv(find_dotenv())
```

Comment:
Add condition to skip tests that use variables from the env file it it does not exist